### ISSUE TITLE
PayPal 2FA Login Not Working

### ISSUE THREAD

**Issue by parose42** [thenbrent/paypal-digital-goods#15]
User's using PayPal's 2FA (two factor authentication) security can't login using the Digital Goods API.  They basically get a user/password combination error on every login request.  As this is a growing feature that's being actively promoted by PayPal to improve user security, it would be good to support if possible.

**Comment by iadj** [comment]
Has this been fixed?

**Comment by thenbrent** [comment]
@iadj no, I can't fix it if I can't get any information about it. :smile: 

If you can send me a link to documentation of PayPal's 2FA or some other source discussing what it is and how it's implemented, I can look into it further.
